### ISSUE TITLE
Let’s add FuseBox

### ISSUE THREAD

**Issue by nchanged** [TheLarkInn/bundler-performance-benchmark#7]
Hey Sean, nice benchmark, I also feel that’s its not fair to show some numbers out of thin air, claiming to be “blazing fast” which is fuse motto btw xD

You think I can dig in and add fuse benchmark there?
 
UPD:
I think it's fair to let parcel and fuse-box to use cache, as well as not uglifying the output. Besides, production builds are made differently as opposed to development flow. That's what i got

```
parcel: 67.12ms | 50.58 KiB
fuse-box: 11.04ms | 44.10 KiB
webpack: 81.93ms | 5.90 KiB
```

UPD2:
Cache off, for fuse-box and parcel

```
fuse-box: 42.44ms | 36.74 KiB
parcel: 161.83ms | 38.97 KiB
webpack: 61.63ms | 9.70 KiB
```

**Comment by TheLarkInn** [comment]
I would love to see that! :-D
Would you like to make a PR? I know the repo format is not _perfect_ but hey that's open source!

**Comment by nchanged** [comment]
@TheLarkInn yep, I can do that. You mind if I refactor the launcher? It would be cool to launch all benchmarks at once. 

Also, i have a question about caching, is it possible to enable caching in webpack? I mean, I would like to make fair benchmarks with different approaches, like cache off and cache on, but I can't just add webpack everywhere to these benchmarks cuz there is no flag to toggle ;-) 

Btw, webpack surprised me, great speed improvements

**Comment by TheLarkInn** [comment]
Please by all means refactor away. I built this on a live stream POC. And I'd love to see it flourish and become easier and more accessible for any bundler to add their benches.
